software/building-android/en.md

```diff
@@ -121,6 +121,31 @@ export CCACHE_COMPRESS=1
 
 *After making changes to your `~/.bashrc` file, remember to run `source ~/.bashrc` for changes to take effect.*
 
+## ADB/Fastboot
+
+You want to install "Android Tools" from the third party software center.
+
+### ADB udev rules
+``` bash
+sudo curl --create-dirs -L -o /etc/udev/rules.d/51-android.rules -O -L https://raw.githubusercontent.com/M0Rf30/android-udev-rules/master/51-android.rules
```

Comment:
I do not feel comfortable with having people curl an arbitrary file onto their system and would much prefer more verbose instructions detailing the creation of the directory, as well as providing the rules file contents in the article for them to add.